.github/workflows/gitlab-ci.yaml

```
@@ -144,15 +144,15 @@ jobs:
+          python manage.py runserver 0.0.0.0:8080
 
       - name: Set up Go
         uses: actions/setup-go@v4
         with:
-          go-version: '1.20'
+          go-version: '1.21'
 
       - name: Run Go script
-        run: go run SystemTest/main.go
\ No newline at end of file
+        run: cd SystemTest && go run main.go
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)